.circleci/config.yml

```diff
@@ -125,6 +125,7 @@ jobs:
       - *restore_ruby_cache
       - *install_ruby_dependencies
       - *save_ruby_cache
+      - run: cd decidim-$CIRCLE_JOB && bundle install && cd ..
```

Comment:
I _think_ we can _replace_ the `install_ruby_dependencies` step with this. If the plugin does have a specific `Gemfile`, `bundler` will use it, otherwise, it will use the root `Gemfile`. Also, I think `run` uses a separate subshell so you don't need to go back with `cd ..`.

Reply:
Let me work on that on a different PR, so we unblock `master` and people can keep working...

Reply:
Yeah, absolutely, unblocking master is the priority!